### PR TITLE
Badge component

### PR DESCRIPTION
**Pull request by dev-prabhat**

### CODE REVIEW

File: document/badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  padding: 8px;
+  box-shadow: 0px 0px 15px 0px var(--primary-theme-light-color);
+}
+
+.grid-container {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  grid-template-areas:
+    "header header header"
+    "side badges badges";
```
Comment (zoef-yt) [inline document/badges/badges.css:16]: Can be written
header header
  side    badges

**Review by developer-anand — APPROVED** [review]
Great Work.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: document/badges/badges.css (hunk 1 above)
Comment (dev-prabhat) [reply document/badges/badges.css:16]: I have divided the whole page in 3 column , give side 1/3 and badges 2/3

Comment (zoef-yt) [reply document/badges/badges.css:16]: it can be divided  into 2 columns
1 with less space and second with 1fr

Comment (dev-prabhat) [reply document/badges/badges.css:16]: But then I have to give width to side section

Comment (zoef-yt) [reply document/badges/badges.css:16]: yes you can have more control over it

Comment (dev-prabhat) [reply document/badges/badges.css:16]: yeah but I want that it takes width automatically with respect to content in that section

**Review by zoef-yt — APPROVED** [review]

**Comment by archit7448** [comment]
looks good to merge

**Review by archit7448 — APPROVED** [review]

**Pull request merged**